package.json

```diff
@@ -8,6 +8,7 @@
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
     "@microlink/react-json-view": "^1.22.2",
+    "@storybook/preview-api": "^8.1.10",
```

Comment:
This should actually be in the dev section of dependencies. I'm adding it in https://github.com/DataBiosphere/terra-ui/pull/4923.